components/server/src/container-module.ts

```diff
@@ -332,11 +328,7 @@ export const productionContainerModule = new ContainerModule(
         bind(RelationshipUpdater).toSelf().inSingletonScope();
 
         // grpc / Connect API
-        bind(APIUserService).toSelf().inSingletonScope();
-        bind(APITeamsService).toSelf().inSingletonScope();
-        bind(APIWorkspacesService).toSelf().inSingletonScope();
-        bind(APIStatsService).toSelf().inSingletonScope();
-        bind(API).toSelf().inSingletonScope();
+        API.contribute(bind);
```

Comment:
So far we often called these methods `abcBinder` or `bindXyz`, bot contribute works for me.